### ISSUE TITLE
Cannot activate lower thirds after OBS update 27.2

### ISSUE THREAD

**Issue by MP5Squeaky** [noeal-dac/Animated-Lower-Thirds#21]
Red border when activating lower thirds after OBS 27.2 update. Worked around by reverting to OBS version 27.1.3

![LTbug](https://user-images.githubusercontent.com/47050508/154572454-5ee7170f-61db-4c65-9ef8-79b25256e9bb.png)

OS Name	Microsoft Windows 10 Pro
Version	10.0.19043 Build 19043

 - OBS version 27.2
 - Version 1.6

**Comment by MP5Squeaky** [comment]
Possibly related to https://github.com/obsproject/obs-studio/issues/5932

**Comment by MP5Squeaky** [comment]
Likely fix:
https://obsproject.com/forum/threads/animated-lower-thirds-with-dockable-control-panel.127896/post-562015

**Comment by Mazecreator** [comment]
Here is a quick video I put together to help get the Animated Lower Thirds working again:

https://www.youtube.com/watch?v=Gbvr3u_NjDg

**Comment by jannessm** [comment]
TL;DR:

don't use local file for the source browser. Just add `file://` before your path to the `browser-source.html`
